src/Documentation/glossary.js

```diff
@@ -13,8 +13,8 @@ code, ML models, etc. A workspace becomes a **DVC project** when
 [\`dvc init\`](/doc/commands-reference/init) is run, and
 [DVC-files](/doc/user-guide/dvc-file-format) or stage files are created in it.
 
-Note that [external outputs](/doc/user-guide/external-outputs) also form part
-of your expanded workspace, technically.
+Note that [external outputs](/doc/user-guide/external-cache-and-outputs) also
```

Comment:
I wonder if we can create a script that checks every link as a test on CI system? :)

Reply:
> I wonder if we can create a script that checks every link as a test on CI system? :)

It is easy to create a script that extracts all the URL-s from a file. Also it is easy to check each of them if it exists. I am not sure about how to use this on a CI system.

Can we create an issue for implementing this?

There is also this: https://wummel.github.io/linkchecker/ But it seems to be a bit old and bloated.

Reply:
Yeah good idea. Who implemented the initial CI pipeline and yarn tests? Let's ask them for now (: Opened https://github.com/iterative/dvc.org/issues/652, please tag them there.